core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/io/InputImplTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2016 Crown Copyright
```

Comment:
Nope, I didn't create that class. The date remains from when it was created correct?